js/zh-cn.js

```diff
@@ -0,0 +1,172 @@
+var i18n = {
+error1: '抱歉，启动原理图工具时出现浏览器错误。我们建议使用最新版本的Firefox和Chrome。',
+Ground_connection: '接地连接',
+Node_label: '节点标签',
+Voltage_source: '电压源',
+Current_source: '电流源',
+Resistor: '电阻',
+Capacitor: '电容',
+Inductor: '电感',
+Op_Amp: '运算放大器',
+Diode: '二极管',
+NFet: 'N沟道MOS管',
+PFet: 'P沟道MOS管',
+Voltage_probe: '电压探头',
+Current_probe: '电流探头',
+drag_onto_diagram: ': 拖动或点击插入',
+Help: '帮助：显示帮助页面',
+Grid: '网格：切换网格显示',
+Link_tip: '链接：分享电路的链接',
+Cut: '剪切选定的组件到剪贴板',
+Copy:'将选定的组件复制到剪贴板',
+Paste:'将剪贴板粘贴到原理图上',
+Delete:'删除选定的组件',
+Rotate:'旋转选定的组件',
+Save_netlist: '保存网表',
+Open_netlist: '打开网表',
+Select_netlist: '选择网表',
+Perform_DC_Analysis: '执行直流分析',
+DC_Analysis: '直流分析',
+Perform_AC_Analysis: '执行交流小信号分析',
+Perform_Transient_Analysis: '执行瞬态分析',
+Transient_Analysis: '瞬态分析',
+Edit_Properties: '编辑属性',
+Link: '链接',
+Sharable_Link: '分享链接',
+
+points_per_decade: 'Number of points/decade',
+Starting_frequency: '开始频率 (Hz)',
+Ending_frequency: '结束频率 (Hz)',
+source_for_ac: '需要交流分析的电压或电流源的名字',
+AC_Analysis_add_a_voltage_probe: '交流分析：在电路图中添加一个电压探头！',
+AC_Analysis: '交流分析',
+Zero_ac_response: 'Zero ac response，使用DB衡量无穷大',
```

Comment:
Full translation? 
Candidate from Google Translate: 接近零交流响应，删除

Reply:
Could you please give me an example about **Zero ac response** or a definition? I cant fully understand it .

Reply:
There are two related warnings when the AC response is zero or close to zero. 
The phrases are very compressed English, and I can see how it might not make sense.

See line 3138 of schematic.js
The phrase "Zero ac response, -infinity on DB scale." really means "The AC response is zero. (which you know is -infinity on the dB scale)" (actually < 10^-16)

The other related phrase is, (line 3147 in schematic.js)
"Near zero ac response, remove " can be expanded to "The AC response is nearly zero. Remove 
 ...". This phrase is followed by the color of the problem probe. AC response is "nearly zero" when it is < 10^-10.

Reply:
Get it.

Google translate is right. I had misunderstanded it before. I will change it later.